meetup/tests.py

```diff
@@ -47,65 +91,62 @@ def test_meetup_link_is_unique(self):
     self.assertRaises(IntegrityError, n.save)
 
   def test_getting_all_associated_pizzas(self):
-    meetup= Meetup.objects.create(name="Meeetup1", meetup_link='http://meetup.com/some-meetup')
-    place = meetup.pizza_places.create(name="Pete Zazz")
-    self.assertEquals(place, meetup.pizza_places.all()[0])
+    meetup= Meetup.objects.create(name="Meetup1", meetup_link='http://meetup.com/some-meetup')
+    place = meetup.pizza_places.create(name="Pete Zazz", yelp_link='https://www.yelp.com/biz/prince-st-pizza-new-york')
+    self.assertEquals(place, meetup.pizza_places.first())
 
 class TestMeetupModelValidations(TestCase):
 
   def setUp(self):
-    self.patcher = patch('meetup.models.MeetupApiLookupAgent')
+    self.patcher = patch('meetup.models.MeetupApi')
     self.mock_agent = self.patcher.start()
-    self.addCleanup(self.patcher.stop)
 
   def test_meetup_raises_error_on_invalid_url(self):
-    meetup= Meetup(name="Meeetup1", meetup_link='hi/ok/what')
+    meetup= Meetup(name="Meetup1", meetup_link='hi/ok/what')
     self.assertRaises(ValidationError, meetup.full_clean)
 
   def test_error_raised_if_link_does_not_point_to_meetupdotcom(self):
-    meetup= Meetup(name="Meeetup1", meetup_link='http://www.example.com/')
+    meetup= Meetup(name="Meetup1", meetup_link='http://www.example.com/')
     self.assertRaises(ValidationError, meetup.full_clean)
 
   def test_error_raised_if_no_urlname_in_meetup_url(self):
```

Comment:
I don't see tests for the error messages around these validations. How were you able to add them to the production code?